doc/developer-documentation.md

```diff
@@ -5,7 +5,7 @@ description: "KernelCI Pipeline developer manual"
 weight: 4
 ---
 
-## Enabling a new Kernel tree
+## Enabling new Kernel trees and tests
```

Comment:
How does the following sound like? We are enabling builds as well.
```suggestion
## Enabling new Kernel trees, builds and tests
```

Reply:
Sounds good.